packages/@tailwindcss-cli/src/commands/build/index.ts

```diff
@@ -141,10 +141,6 @@ export async function handle(args: Result<ReturnType<typeof options>>) {
         return require(pluginPath)
       },
       onContentPath(glob) {
-        if (glob[0] !== '/') {
-          glob = path.join(basePath, glob)
-        }
-
```

Comment:
Something to figure out in the morning @philipp-spiess 

But deleting this makes the test fail again, the reason being is that each `glob` is relative to the `input.css` file (or whatever the input is). The `input.css` file lives in `basePath`. But we use `base` instead of `basePath` when calling the `scanDir` function.

The reason we use `base` (typically the `process.cwd()`) is so that auto content detection can be executed from where you execute your command.

Reply:
@RobinMalfait Ah right, the two different `base`es, I remember now. 😐 

I think instead of checking for `if (glob[0] !== '/') {` we should then invert it and only append the `basePath ` for relative globs (so those starting with `./` or `../`). This should work on Windows as well since we rewrite globs with posix seperators and we use the same rule for detecting relative ones in the `postcss-fix-relative-paths` plugin.

Also maybe while we're sweeing around here, what about we keep `base` for the `process.cwd()` equivalent and use `stylesheetPath` or so for the root css path?